packages/react/src/components/Text/index.stories.js

```diff
@@ -11,7 +11,7 @@ storiesOf("Text", module)
       <Text>Default Text</Text>
 
       <strong>
-        <Text>Strong Text</Text>
+        <Text as="strong">Strong Text</Text>
```

Comment:
Confused what this does: why does it send `as="strong"` when is already inside a `<strong>` tag?

Reply:
🤦‍♂ fixed